### PR TITLE
Fix grammar issues and multiline strings

### PR DESCRIPTION
**Pull request by BasedUser**
So. Many. Strings.

Please review this, as I'm not confident I've got the spirit down.

### CODE REVIEW

File: src/main/resources/bundles/bundle.properties [hunk 1]
```diff
@@ -1,211 +1,150 @@
...
 
-name-modified = Your name hed illegal sequences that were aromatically removed.
+name-modified = Your name hed illegal characters that were automatically removed.
 
-show-modes = Available modes are : {modes}
+show-modes = Available modes: {modes}
 
-pet-invalid-trail = pet error in {pet-name} : Trail name {provided} is invalid, using default fire.
-pet-invalid-bullet = pet error in {pet-name} : Bullet name {provided} is invalid, shooting on the pet is disabled.
+pet-invalid-trail = pet error in {pet-name}: Trail name {provided} is invalid, using default shots.
```
Comment (jakubDoka) [inline src/main/resources/bundles/bundle.properties]: fire is default trail.

File: src/main/resources/bundles/bundle.properties [hunk 2]
```diff
@@ -1,211 +1,150 @@
...
-test-help = Test is here for automatic [sky]verification[]. To pass you have to answer [orange]all[] questions correctly. \
-  If you fail you will get [orange]15[] min [scarlet]penalty[].\n\
-[orange]/test start[] - Starts the test.\n\
-[orange]/test egan[] - If you forgot the questions you can ask egan.\n\
-[orange]/test <answer>[] - Argument has to be number and it has to be one of the options.
-
-set = Settings :
+test-is-recent = You recently took the test, try again in {time}.
+test-no-need = You don't need to be tested as you are already {verified}.
+test-help = Taking the test is important if you want to [sky]verify[] yourself. To pass you have to answer [orange]all[] questions correctly. \nIf you fail you will need to wait [orange]15[] minutes as a [scarlet]penalty[].\n[orange]/test start[] - Starts the test.\n[orange]/test egan[] - Print the last question asked again.\n[orange]/test <answer>[] - Answer with option <answer>.
```
Comment (jakubDoka) [inline src/main/resources/bundles/bundle.properties]: This is just my personal preference i do not like long lines

File: src/main/resources/bundles/bundle.properties [hunk 3]
```diff
@@ -1,211 +1,150 @@
...
...ast 100 buildings, and that the player must have at least 10x more buildings built than the amount of\nhours he played, so if you built 10 buildings every hour you will have the rank. Any less and that rank will be removed.\n[stat]More examples:[] "quests":{"buildingsBroken":{best:10},"playtime":{required:1000 * 60 * 60 * 100}};\n"quests":{"messageCount":{"frequency":100,"bets":3,"required":10000}}.\nList of available stats:\n{stats}\nIf you still don't understand this system, open an issue on github.\nlinked - If you already defined a quest in a rank and want to use them in another, this property can be used. Lets say\nyou defined bulldozer and kamikaze rank and want to combine their requirements into one rank's requirements. That rank\nwould need to have "linked":["kamikaze","bulldozer"] as its property. That rank will be unobtainable if a nonexistent\nrank is linked to.\npets - Pets are defined as a JSON array: "pets":["foo","bar"]. This array will spawn 2 pets attracted to the\nplayer. If the pets "foo" or "bar" don't exist, this won't work and you will get an error whenever a player with that\nrank joins.\nvalue - The priority value decides which rank should be rendered first.
```
Comment (jakubDoka) [inline src/main/resources/bundles/bundle.properties]: egain

File: src/main/resources/bundles/bundle.properties [hunk 4]
```diff
@@ -1,211 +1,150 @@
...
+chat-disabled = You have the chat disabled. To enable chat, write [orange]/set chat on[].
 
-warming-spam = {} more quick uses of command and you will be kicked and banned.
+warming-spam = {} more uses of command in a row and you will be banned.
 
 hud-core-under-attack = !!CORE IS UNDER ATTACK!!
 
-discord-pending-link = Discord user [orange]{user-name}[] is trying to link the accounts,if that's you, use \
-[green]/link <your pin bot gave you to confirm liking>[] or [scarlet]/link refuse[] fi isn't.
+discord-pending-link = Discord user [orange]{user-name}[] is trying to link this account. If it's you, use \n[green]/link <your pin bot gave you to confirm>[] or if not, [scarlet]/link refuse[].
```
Comment (jakubDoka) [inline src/main/resources/bundles/bundle.properties]: another one

File: src/main/resources/bundles/bundle.properties [hunk 5]
```diff
@@ -1,211 +1,150 @@
...
+vote-in-process = Wait until current vote ends.
+vote-not-active = Nothing to vote for.
 vote-y-n = Write [green]"y"[] or [scarlet]"n"[] to the chat.
 vote-/vote-y-/vote-n = Use [green]/vote y[] or [scarlet]/vote n[].
 vote-req = requires
-vote-cannot-vote = Wait until new game starts please.
+vote-cannot-vote = Wait until a new game starts, please.
 
-mkgf-self = You cannot mark your self a griefer.
-mkgf-target-admin = Do you really expect to be able to mark admin?
```
Comment (jakubDoka) [inline src/main/resources/bundles/bundle.properties:164]: i like this one better

File: src/main/resources/bundles/bundle.properties [hunk 6]
```diff
@@ -1,211 +1,150 @@
...
+map-help = \n[orange]/map change <name/idx>[] - Opens a vote to change map to specified one. You can figure out index from [orange]/map list[],\n[orange]/map gameover[] - When you know you will lose, you can end the game with this command.\n[orange]/map restart[] - Restarts the game.\n[orange]/map list <idx>[] - Shows list of maps divided to pages.\n[orange]/map info [name/idx][] - Shows info about map you playing or about map you specify.\n[orange]/map rate <name/idx/this> <1-10>[] - You can give map a rating. You can specify witch map or just write [orange]this[] to rate map you are playing.
```
Comment (jakubDoka) [inline src/main/resources/bundles/bundle.properties]: once egan this is not readable

**Review by jakubDoka — CHANGES_REQUESTED** [review]
Looks priti fine good grammar practice for me too just thease long lines are messy.

File: src/main/resources/bundles/bundle.properties (hunk 5 above)
Comment (jakubDoka) [reply src/main/resources/bundles/bundle.properties:164]: i mean mkgf-target-admin one.

File: src/main/resources/bundles/bundle.properties (hunk 2 above)
Comment (BasedUser) [reply src/main/resources/bundles/bundle.properties:43]: They are unfortunately required for Mindustry to detect them as one continuous line.

**Comment by BasedUser** [comment]
Unfortunately, these long lines are REQUIRED for mindustry to even consider them as valid.

E: Reread the documentation. No. Reupdating.

**Pull request merged**